enioka_scan/src/main/java/com/enioka/scanner/sdk/bluebird/BluebirdScanner.java

```diff
@@ -63,6 +63,21 @@ public void onReceive(Context context, Intent intent) {
     }
 
 
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    // SOFTWARE TRIGGERS
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void pressScanTrigger() {
+        // FIXME
```

Comment:
Not really a fixme nor a todo... just not implemented because likely not supported.